src/js/controllers/index.js

```diff
@@ -406,14 +406,33 @@ angular.module('copayApp.controllers').controller('indexController', function($r
     var config = configService.getSync().wallet.settings;
     self.pendingTxProposalsCountForUs = 0;
     lodash.each(txps, function(tx) {
-      var amount = tx.amount * self.satToUnit;
-      tx.amountStr = profileService.formatAmount(tx.amount) + ' ' + config.unitName;
-      tx.feeStr = profileService.formatAmount(tx.fee) + ' ' + config.unitName;
-      tx.alternativeAmount = rateService.toFiat(tx.amount, config.alternativeIsoCode) ? rateService.toFiat(tx.amount, config.alternativeIsoCode).toFixed(2) : 'N/A';
-      tx.alternativeAmountStr = tx.alternativeAmount + " " + config.alternativeIsoCode;
-      tx.alternativeIsoCode = config.alternativeIsoCode;
+      function formatAmount(obj, amount) {
+        obj.amountStr = profileService.formatAmount(obj.amount) + ' ' + config.unitName;
+        obj.alternativeAmount = rateService.toFiat(obj.amount, config.alternativeIsoCode) ? rateService.toFiat(obj.amount, config.alternativeIsoCode).toFixed(2) : 'N/A';
+        obj.alternativeAmountStr = obj.alternativeAmount + " " + config.alternativeIsoCode;
+      };
 
+      if (tx.outputs) {
+        tx.amount = lodash.reduce(tx.outputs, function(total, o) {
```

Comment:
It could be simpler to have: 
`tx.summary`
and
`tx.formatedOutputs`
separated.

`summary` could have the accumulated amount, and the recipient count.
`formatedOutputs` could by a list with already formatted amounts and note. 

Then, the template would show the multioutput variant if `summary` exists, and if expanded, formatedOutput will be shown (not need to call expand, but only set a template variable).